src/reference/interactive/index.pug

```diff
@@ -318,26 +334,26 @@ block reference
 
     p A method can contain parameters which get provided to the recipient.
 
-    p Methods contain an additional property called #[code discard] which when true indicates that the recipient can choose not to respond. Methods that can be treated as events have the #[code discard] property set to true.
+    p Methods contain an additional property called #[code discard], which when TRUE, indicates that the recipient can choose not to respond. Methods that can be treated as events have the #[code discard] property set to TRUE.
```

Comment:
lets go with ``true``, booleans are lowercase in just about every language, being uppercase makes me think we're writing SQL or something